kernel/packages/voice-chat-codec/VoiceCommunicator.ts

```diff
@@ -317,20 +316,26 @@ export class VoiceCommunicator {
 
       const offer = await src.createOffer()
 
-      await src.setLocalDescription(offer)
+      await src.setLocalDescription(offer).catch((err) => {
+        return Promise.reject(err)
```

Comment:
The following two snippets are exactly the same:

```ts
await src.setLocalDescription(offer)
```

```ts
await src.setLocalDescription(offer).catch((err) => {
  return Promise.reject(err)
})
```

We should remove the `.catch` to avoid confussions